OS/Linux/ABI_libc.h

```diff
@@ -0,0 +1,291 @@
+// 
+// Tests for generating ABI library functions
+// Include all (almost all?) files from /usr/include
+//
+// TODO: automate running this :)
```

Comment:
I agree, I want to make this work. Right now one issue I have is that some symbols detected by cparser are not valid when building with the clang from cxx-common (different defines, etc), so the file required a few manual edits.

Reply:
To avoid missing headers issue, temporarily we can add following packages in the build scripts:
`comerr-dev, libgcrypt11-dev, zlib1g-dev, libkrb5-dev, libhwloc-dev, libldap2-dev, liblz4-dev, liblzma-dev, libmagic-dev, libmpc-dev, libwrap0-dev, libuv1-dev, iptables-dev`

Reply:
Is it preferable to add these dependencies, or to go and just generate the ABI files from a fresh ubuntu install?